src/app.ts

```diff
@@ -0,0 +1,77 @@
+import dotenv from 'dotenv';
+import express from 'express';
+import morgan from 'morgan';
+import { join } from 'path';
+import { Writable } from 'stream';
+import { SpreadSheet } from '../spreadsheets/spread-sheet';
+import { spreadSheetUpdater } from '../spreadsheets/spread-sheet-updater';
+
+if (process.env.NODE_ENV === 'local') {
```

Comment:
Usually, you don't create special `NODE_ENV` value for local.
Usually we differentiate between `production` and all the rest

Reply:
I have preloaded the dotenv package which does the configuration on local env. So I remove these lines.